elab-business/elab-business-impl/src/main/java/com/tolean/elab/business/impl/dictionary/DictionaryServiceImpl.java

```diff
@@ -32,4 +36,29 @@ public DictionaryViewDto getDictionaries(String code) {
     return dictionaryMapper.toDictionaryViewDto(dictionary);
   }
 
+  @Override
+  public DictionaryViewDto add(DictionaryNewDto dictionaryNewDto){
+    checkNotNull("20170206:1629", dictionaryNewDto);
+
+    Dictionary dictionary = dictionaryMapper.toDictionary(dictionaryNewDto);
```

Comment:
- [ ] tutaj powinno być pobranie Dictionary na podstawie dictionaryCode. Jeśli słownik nie został odnaleziony to rzucenie wyjątkiem DictionaryNotFoundException